http/headers/set-cookie.json

```diff
@@ -170,7 +170,12 @@
                 "version_added": "60"
               },
               "ie": {
-                "version_added": false
+                "version_added": "11",
+                "partial_implementation": true,
+                "notes": [
+                  "Not shipped with the inital release but later with the 2018 June security update (Patch Tuesday) to Windows 10 RS3 (2017 Fall Creators Update) and newer. <a href='https://github.com/MicrosoftEdge/Status/issues/616'>More info</a>.",
+                  "Partial support because only supported in IE 11 on Windows 10 RS3 (2017 Fall Creators Update) and newer, but not in IE 11 on other Windows versions (Windows 7, ...)"
+                ]
```

Comment:
I think that we should omit these notes, and just say `"version_added": "11"` (no partial implementation either).  Even though CanIUse is recording this data, I believe that it's more detail than this project is targeting.  This is the same reasoning as why we [only record major and minor version updates but not patch updates in BCD](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#release-lines), as users are most likely to run the latest patch release for their browser

Reply:
Yes, I understand. I would like to respond.